src/migrator/sync.ts

```diff
@@ -30,7 +30,7 @@ export const postRun = async (tx: Tx, model: ApiRootModel): Promise<void> => {
 			`First time executing '${modelName}', running init script`,
 		);
 
-		await lockMigrations(tx, modelName, async () => {
+		await lockMigrations(tx, modelName, true, async () => {
```

Comment:
I think this is reaching the point where it probably makes sense to use an options object in order to have named parameters, ie
```suggestion
		await lockMigrations({tx, modelName, blocking: true}, async () => {
```

Reply:
Thanks!